server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -59,9 +59,13 @@ impl RequisitionTransferProcessor for CreateResponseRequisitionProcessor {
         // TODO link to documentation of how remote authorisation works
         let store_preference =
             get_store_preferences(connection, &record_for_processing.other_party_store_id)?;
-        let approval_status = store_preference
-            .response_requisition_requires_authorisation
-            .then_some(RequisitionRowApprovalStatus::Pending);
+        let approval_status = if store_preference.response_requisition_requires_authorisation
+            && request_requisition.requisition_row.program_id.is_some()
```

Comment:
Authorisation status should only be set if the requisition is a program requisition..

Reply:
that's a good catch! thanks 😃  

this requirement is a just for DJ though. I'm wondering if we just implement for now but add a comment and rework when we have plugin functionality for the back end?
We don't want it to work this way for all clients. If authorisation is configured then it should be for all requisitions.

Reply:
Hmm we would need to remember to update it in mSupply as well, for omSupply transfers mSupply will only generate authorisation for program requisitions: https://github.com/sussol/msupply/blob/45ebf319e2b862dc46a015a75112173aa8eb2de5/Project/Sources/Methods/requisitionAuthorisationHandler.4dm#L28

Reply:
oh! right. well, that's easier, if that's the base functionality. if we only ever create auth for program requisitions. that'd need documenting though.